cmd/corimDisplay.go

```diff
@@ -61,72 +61,119 @@ func checkCorimDisplayArgs() error {
 	return nil
 }
 
-func display(signedCorimFile string, showTags bool) error {
-	var (
-		signedCorimCBOR []byte
-		metaJSON        []byte
-		corimJSON       []byte
-		err             error
-		s               corim.SignedCorim
-	)
-
-	if signedCorimCBOR, err = afero.ReadFile(fs, signedCorimFile); err != nil {
-		return fmt.Errorf("error loading signed CoRIM from %s: %w", signedCorimFile, err)
+func displaySignedCorim(s corim.SignedCorim, corimFile string, showTags bool) error {
+	metaJSON, err := json.MarshalIndent(&s.Meta, "", "  ")
+	if err != nil {
+		return fmt.Errorf("error encoding CoRIM Meta from %s: %w", corimFile, err)
 	}
 
-	if err = s.FromCOSE(signedCorimCBOR); err != nil {
-		return fmt.Errorf("error decoding signed CoRIM from %s: %w", signedCorimFile, err)
+	fmt.Println("Meta:")
+	fmt.Println(string(metaJSON))
+
+	corimJSON, err := json.MarshalIndent(&s.UnsignedCorim, "", "  ")
+	if err != nil {
+		return fmt.Errorf("error encoding unsigned CoRIM from %s: %w", corimFile, err)
 	}
 
-	if metaJSON, err = json.MarshalIndent(&s.Meta, "", "  "); err != nil {
-		return fmt.Errorf("error decoding CoRIM Meta from %s: %w", signedCorimFile, err)
+	fmt.Println("Corim:")
```

Comment:
Please use consistent capitalization.
```suggestion
	fmt.Println("CoRIM:")
```

Reply:
Resolved...